lib/build/ProjectBuilder.js

```diff
@@ -34,11 +35,12 @@ class ProjectBuilder {
 	 * Executes a project build, including all necessary or requested dependencies
 	 *
 	 * @public
-	 * @param {@ui5/project/graph/ProjectGraph} graph Project graph
-	 * @param {@ui5/builder/tasks/taskRepository} taskRepository Task Repository module
-	 * @param {@ui5/project/build/ProjectBuilder~BuildConfiguration} [buildConfig] Build configuration
+	 * @param {object} parameters
+	 * @param {@ui5/project/graph/ProjectGraph} parameters.graph Project graph
+	 * @param {@ui5/project/build/ProjectBuilder~BuildConfiguration} [parameters.buildConfig] Build configuration
+	 * @param {@ui5/builder/tasks/taskRepository} parameters.taskRepository Task Repository module to use
 	 */
-	constructor(graph, taskRepository, buildConfig) {
+	constructor({graph, buildConfig, taskRepository}) {
```

Comment:
changing the constructor here leads to a breaking change. So commit message should reflect his.